src/mediaelement.js

```diff
@@ -60,6 +60,43 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
         this.setPlaybackRate(this.playbackRate);
     },
 
+    loadElt: function (elt, container, peaks) {
```

Comment:
This method almost entirely duplicates the regular `load` method. Can you share the code between the two methods somehow? Maybe create another function and call it from both methods.